modules/apps/segments/segments-content-targeting-upgrade/src/main/java/com/liferay/segments/content/targeting/upgrade/internal/upgrade/v1_0_0/ContentTargetingUpgradeProcess.java

```diff
@@ -188,7 +190,18 @@ private void _upgradeContentTargetingUserSegments() throws Exception {
 	private static final Log _log = LogFactoryUtil.getLog(
 		ContentTargetingUpgradeProcess.class);
 
-	private final RuleConverterRegistry _ruleConverterRegistry;
+	private static final ServiceTrackerMap<String, RuleConverter>
+		_serviceTrackerMap;
+
+	static {
```

Comment:
Hey @GabrielHenriquedev, instead of initialize the serviceTracker in a static block here, please initialize it in the activate method of `SegmentsContentTargetingUpgradeStepRegistrator` component.